examples/async_https_client.cpp

```diff
@@ -156,7 +159,7 @@ int main(int argc, char** argv) {
   const std::string host = std::string(what[1]);
   // Use default HTTPS port (443) if not specified
   const std::string port = what[2].length() > 0 ? what[2].str() : "443";
-  // Use deault path ('/') if not specified
+  // Use default path ('/') if not specified
```

Comment:
Thanks for fixing my silly typos. To be a bit pedantic it ought to be a separate commit but it's not really that important :-)

Reply:
Credit goes to Visual Assist's spell checking ;-)
Feel free to cherry pick the fixed typos into the master. It did not seem to be worth a separate PR.

Reply:
It's fine. I'll just merge the pull request as it is.